include/msgbuffer.h

```diff
@@ -0,0 +1,82 @@
+/**
+ * @brief MAVConn message buffer class (internal)
+ * @file msgbuffer.h
+ * @author Vladimir Ermakov <vooon341@gmail.com>
```

Comment:
This file is trivial - copying it from another codebase and bringing the whole copyright shebang over is almost excessive. It's good to attribute the original author, but we need to cleanup the copyright notices here - e.g. does OSRF have copyright? I suspect its rather Vladimir alone.

Reply:
Yeah we can probably keep the MAVROS licensing instead.

Reply:
Yeah we can probably keep the MAVROS licensing instead.